API/gimvicurnik/__init__.py

```diff
@@ -373,8 +372,36 @@ def _get_documents():
                     for model in query
                 ]
             )
+        @self.app.route("/circular/atom")
+        def _get_atom():
+            query = self.session.query(Document.date, Document.type, Document.url, Document.description).filter(or_(Document.type == "circular", Document.type == "None")).order_by(Document.date)
+
+            config = self.config["sources"]["eclassroom"]["pluginfile"]
+            token = self.config["sources"]["eclassroom"]["token"]
 
+            return render_template("atom.xml", entries = query, last_updated = max(model.date for model in query).strftime("%Y-%m-%d"))
+        @self.app.route("/circular/rss")
+        def _get_rss():
+            query = self.session.query(Document.date, Document.type, Document.url, Document.description).filter(or_(Document.type == "circular", Document.type == "None")).order_by(Document.date)
+
+            config = self.config["sources"]["eclassroom"]["pluginfile"]
+            token = self.config["sources"]["eclassroom"]["token"]
 
+            return render_template("rss.xml", entries = query)
```

Comment:
Malo popravi pa dodaj URLje feede:

* `/feeds/circulars.rss` in `/feeds/circulars.atom` za okrožnice in "ostale" dokumente
* `/feeds/substitutions.rss` in `/feeds/substitutions.atom` za dokumente z nadomeščanji
* `/feeds/schedules.rss` in `/feeds/schedules.atom` za dokumente za razporede kosila
* `/feeds/menus.rss` in `/feeds/menus.atom` za jedilnike za malico in kosilo

V glavnem tako kot so kategorije na spletni strani.

Reply:
sam why
Zakaj rabiš ` /feeds/substitutions.rss `

Reply:
Da bi ti nadomeščanja dal v koledar

Reply:
Da ti tiste linke iz Dropboxa da v RSS